src/core/core.ticks.js

```diff
@@ -80,10 +80,15 @@ module.exports = {
 				numSpaces = Math.ceil(numSpaces);
 			}
 
-			// Put the values into the ticks array
+			var precision = 1;
+			if (spacing < 1) {
+				precision = Math.pow(10, spacing.toPrecision().length - 2);
```

Comment:
Would it be equivalent to replace `.toPrecision()` with `.toString()`? I think that would make it clearer

Reply:
In this case I think they both return equal results. Since the code is trying to find the number of significant digits after the decimal point.